modules/govuk_ci/templates/usr/local/bin/remove_old_jenkins_jobs.sh.erb

```diff
@@ -2,4 +2,5 @@
 
 dirs_to_keep='<%= (@pipeline_jobs.keys + @job_builder_jobs.map { |job| job.split("::").last }).join("\n") %>'
 
-ls -A /var/lib/jenkins/jobs | grep -ivxF "$dirs_to_keep" | xargs rm -rf
+cd /var/lib/jenkins/jobs
```

Comment:
You need a `set -eu` at the top so that if this fails of won't run the rm in the wrong dir.